aws/cloudformation/standalone/gen_ai_curriculum/deploy_gen_ai_curriculum_stack

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env ruby
+require 'aws-sdk-applicationautoscaling'
+require 'colorize'
+require 'optparse'
+require 'open3'
+require 'pycall'
+require_relative './config'
+
+def main
+  options = {}
+  OptionParser.new do |opts|
+    opts.banner = "Usage: ./deploy_gen_ai_curriculum_stack [options]"
+
+    opts.on("-e", "--environment ENVIRONMENT", String, "Stack environment (prod or test)") do |v|
+      options[:environment] = v
+    end
+
+    opts.on("-d", "--dry-run [FLAG]", TrueClass, "Dry run (create template only but don't deploy)") do |v|
+      options[:dry_run] = v.nil? ? true : v
+    end
+
+    opts.on("-v", "--image-uri-version", Numeric, "HuggingFace Image URI version") do |v|
+      options[:image_uri_version] = v
+    end
+
+    opts.on("-i", "--image-uri IMAGE_URI", String, "HuggingFace Image URI to use") do |v|
+      options[:image_uri] = v
+    end
+  end.parse!
+
+  # Prompt for environment if not provided
+  unless options[:environment]
+    print "Environment (prod or test): "
```

Comment:
I chose 'test' as the non-prod environment name kind of arbitrarily, open to other options.

Reply:
There might be a small advantage to using `production` rather than `prod`, just because it would then match our RAILS_ENV. But there are obviously disadvantages took like the names get longer. No strong feelings, just an idea.